python/lsst/daf/butler/cli/opt/repo.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import click
+
+__all__ = ["repo_option"]
+
+
+class repo_option:  # noqa: N801
+    def __init__(self, required=False):
+        self.required = required
+
+    def __call__(self, f):
+        return click.option('--REPO',
```

Comment:
from Jim's document I thought he wanted `REPO`. I agree that `--repo` is better for a CLI option. (I've since discovered that in documentation using all caps for *argument* names is normal, but does not mean that *option* names should be all caps)